tests/src/FunctionalJavascript/SkipToMainContentTest.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_whitelabel\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Verifies the "Skip to main content" link focuses <main id="main-content">.
+ */
+class SkipToMainContentTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'oe_whitelabel';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_bootstrap_theme_helper',
+  ];
+
+  /**
+   * Ensures the skip link focuses the main landmark.
+   */
+  public function testSkipToMainContentFocusesMain(): void {
+    $this->drupalGet('<front>');
+    $assert = $this->assertSession();
+    $skip_link = $assert->elementExists('css', 'a[href="#main-content"]');
+    $this->getSession()->executeScript(<<<'JS'
+      (function () {
+        var a = document.querySelector('a[href="#main-content"]');
+        if (!a) return;
+        a.focus();
+        a.click();
+      }());
+    JS);
+    $hash = $this->getSession()->evaluateScript('return window.location.hash;');
+    $this->assertSame('#main-content', $hash, 'Location hash updated to #main-content.');
+    $active_id = $this->getSession()->evaluateScript('return document.activeElement ? document.activeElement.id : "";');
+    $this->assertSame('main-content', $active_id, 'Focus moved to #main-content.');
+    $main = $assert->elementExists('css', 'main#main-content');
+    $this->assertSame('-1', $main->getAttribute('tabindex'), '<main> has tabindex="-1".');
+    $this->assertCount(1, $this->getSession()->getPage()->findAll('css', 'main#main-content'));
```

Comment:
This seems redundant with earlier assertions.
I guess we want to assert that the element with that id is really the `<main>` element?

Reply:
indeed, it's deleted

Reply:
we already test this on line 45